request.go

```diff
@@ -34,6 +35,12 @@ type Request struct {
 	// response from server is unmarshaled into Result.
 	Result interface{}
 
+	// RawResponseBody can be set to capture the response body for external use.
+	RawResponseBody bool
```

Comment:
Maybe call this bool `CaptureResponseBody` instead?  Just to make it more obvious what the toggle controls.